### ISSUE TITLE
RANDR panning and nxagent

### ISSUE THREAD

**Issue by sunweaver** [ArcticaProject/nx-libs#136]
Hi Uli, hi all,

I would like to bring up another topic regarding RANDR here. The RANDR 1.3 proto version bring "panning" to the Xserver. For a little reader on panning, see here:
https://sfxpt.wordpress.com/2011/02/02/panning-using-xrandr/

I think we should support that for nxagent in fullscreen mode. For this, we need more modes then the current resolution (nx_<phys-x>x<phys-y>) to be available to xrandr.

Is it ok, if I work on this?

**Comment by uli42** [comment]
Yes, it is okay. Does that collide with the viewport stuff nx is already capable of?

**Comment by sunweaver** [comment]
HI Ulrich,

On  Mo 04 Jul 2016 20:27:20 CEST, Ulrich Sibiller wrote:

> Yes, it is okay. Does that collide with the viewport stuff nx is  
> already capable of?

I don't think it does.

Maybe panning does not make much sense in windowed desktop mode (and  
it may also overlap with the viewport stuff), but in fullscreen mode,  
the panning API should be supported in nx-libs, I think. The panning  
always involves resolution changes of the underlying X.org screen, so  
maybe the panning request must be piped through to the underlying  
(client-side) X-server.

Mike

## 

DAS-NETZWERKTEAM
mike gabriel, herweg 7, 24357 fleckeby
mobile: +49 (1520) 1976 148
landline: +49 (4354) 8390 139

GnuPG Fingerprint: 9BFB AEE8 6C0A A5FF BF22  0782 9AF4 6B30 2577 1B31
mail: mike.gabriel@das-netzwerkteam.de, http://das-netzwerkteam.de
